### ISSUE TITLE
ImportError: No module named '__version__'

### ISSUE THREAD

**Issue by Chaz6** [m3nu/upodder#12]
```
$ pip install --user upodder
Collecting upodder
  Downloading upodder-0.6.tar.gz
Requirement already satisfied (use --upgrade to upgrade): clint in /home/chaz/.local/lib64/python3.4/site-packages (from upodder)
Requirement already satisfied (use --upgrade to upgrade): requests in /home/chaz/.local/lib64/python3.4/site-packages (from upodder)
Collecting feedparser (from upodder)
  Downloading feedparser-5.2.1.zip (1.2MB)
    100% |████████████████████████████████| 1.2MB 562kB/s
Collecting sqlobject (from upodder)
  Downloading SQLObject-2.1.2.tar.gz (888kB)
    100% |████████████████████████████████| 890kB 597kB/s
Requirement already satisfied (use --upgrade to upgrade): args in /home/chaz/.local/lib64/python3.4/site-packages (from clint->upodder)
Collecting FormEncode>=1.1.1 (from sqlobject->upodder)
  Downloading FormEncode-1.3.0.zip (347kB)
    100% |████████████████████████████████| 348kB 404kB/s
Building wheels for collected packages: upodder, feedparser, sqlobject, FormEncode
  Running setup.py bdist_wheel for upodder ... done
  Stored in directory: /home/chaz/.cache/pip/wheels/d2/c2/43/8445d32279fc05769c0aa02edfa327db5a54ccf9b906ed8cb2
  Running setup.py bdist_wheel for feedparser ... done
  Stored in directory: /home/chaz/.cache/pip/wheels/15/ce/10/b500f745822ea6db6ea8ed225c06b15c000d71016b89ef9037
  Running setup.py bdist_wheel for sqlobject ... done
  Stored in directory: /home/chaz/.cache/pip/wheels/ce/bb/3c/d85c24595ac5767bc000b054bd1e1e0fbeb430c6945953921c
  Running setup.py bdist_wheel for FormEncode ... done
  Stored in directory: /home/chaz/.cache/pip/wheels/89/df/07/2bb5e9f2c89f5dfa839a366f4148cfd1ee9bc9c661b7160cc6
Successfully built upodder feedparser sqlobject FormEncode
Installing collected packages: feedparser, FormEncode, sqlobject, upodder
Successfully installed FormEncode-1.3.0 feedparser-5.2.1 sqlobject-2.1.2 upodder-0.6
$ upodder
Traceback (most recent call last):
  File "/home/chaz/.local/bin/upodder", line 7, in <module>
    from upodder.upodder import main
  File "/home/chaz/.local/lib/python3.4/site-packages/upodder/upodder.py", line 15, in <module>
    from sqlobject import SQLObject, sqlite, DateTimeCol, UnicodeCol
  File "/home/chaz/.local/lib/python3.4/site-packages/sqlobject/__init__.py", line 2, in <module>
    from __version__ import version, version_info
ImportError: No module named '__version__'
```

**Comment by m3nu** [comment]
Sorry this happened to you, @Chaz6 .

This error happens in `sqlobject` which isn't Python3 compatibale yet and you're using Python 3.4

I'll see if `sqlobject` can be replaced with a more Python 3-compatible package.

Thanks again for reporting this.

**Comment by m3nu** [comment]
Added a different dependency for `sqlobject` that also works on Python 3. Needs testing, but should solve your issue.

**Comment by m3nu** [comment]
Fixed in e0ddee72

**Comment by Chaz6** [comment]
Thank you!
